bot/exts/help_channels/_cog.py

```diff
@@ -267,6 +267,10 @@ async def init_available(self) -> None:
             for channel in channels[:abs(missing)]:
                 await self.unclaim_channel(channel, closed_on=_channel.ClosingReason.CLEANUP)
 
+        self.available_help_channels = {
+            c for c in self.available_category.channels if not _channel.is_excluded_channel(c)
+        }
```

Comment:
I think you could use `_channel.get_category_channels` here instead
https://github.com/python-discord/bot/blob/4c9405b21341880d9bdc1f0dde02219de9ade005/bot/exts/help_channels/_channel.py#L32-L39

Reply:
Yes 052cead